### PR TITLE
Security issues fix

### PR DESCRIPTION
**Pull request by ziveo**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Make sure you are requesting to pull a topic/feature/bugfix branch (right side). Don't request your master!
- [ ] Make sure you are making a pull request against the develop branch (left side). Also you should start your branch off our develop.
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.

## Pull request type

 

 

Please check the type of change your PR introduces:
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -110,6 +109,10 @@
     "webpack-hasjs-plugin": "^1.0.3",
     "webpack-merge": "^4.2.2"
   },
+  "overrides": {
```
Comment (davidlstarr) [inline package.json:112]: @ziveo Why are the overrides needed?

File: package.json [hunk 2]
```diff
@@ -88,14 +87,14 @@
     "eslint-plugin-react-hooks": "^2.3.0",
     "file-loader": "^6.2.0",
     "html-loader": "^0.5.5",
-    "html-webpack-plugin": "^5.6.0",
+    "html-webpack-plugin": "^4.5.2",
```
Comment (davidlstarr) [inline package.json:90]: @ziveo Why was this downgraded instead of upgraded?

File: package.json (hunk 1 above)
Comment (ziveo) [reply package.json:112]: Libraries that are using old versions of `node-forge` and `glob-parent` are not updated for long time and overrides is way to force installations of libraries that dont have security vulnerability.

Same approach I used for GFW Pro project.

File: package.json (hunk 2 above)
Comment (ziveo) [reply package.json:90]: New version of library is conflicting with Webpack v4 and works fine with old version.

html-webpack-plugin

No issues with clean installs.

Screenshot 2024-03-04 at 2 04 27 PM

**Review by davidlstarr — APPROVED** [review]

**Pull request merged**